From: Bram Moolenaar <Bram@vim.org>
Date: Sun, 25 Nov 2018 05:06:48 +0100
Subject: patch 8.1.0547: modeline test with keymap still fails

Problem:    Modeline test with keymap still fails.
Solution:   Check that the keymap feature is available for the failure assert.

(cherry picked from commit 3067a4dd0d768d1e4a47cf3c100f3e2b462717d7)

Signed-off-by: James McCoy <jamessan@debian.org>
---
 src/testdir/test_modeline.vim | 3 +++
 src/version.c                 | 2 ++
 2 files changed, 5 insertions(+)

diff --git a/src/testdir/test_modeline.vim b/src/testdir/test_modeline.vim
index d29bbb5..e0f97c4 100644
--- a/src/testdir/test_modeline.vim
+++ b/src/testdir/test_modeline.vim
@@ -87,5 +87,8 @@ func Test_modeline_syntax_fails()
 endfunc
 
 func Test_modeline_keymap_fails()
+  if !has('keymap')
+    return
+  endif
   call s:modeline_fails('keymap', 'keymap=evil$CMD')
 endfunc
diff --git a/src/version.c b/src/version.c
index 02a53e8..9752cb7 100644
--- a/src/version.c
+++ b/src/version.c
@@ -1195,6 +1195,8 @@ static int included_patches[] =
  */
 static char *(extra_patches[]) =
 {   /* Add your patch description below this line */
+/**/
+    "8.1.0547",
 /**/
     "8.1.0546",
 /**/
